templates/aws/pro.html

```diff
@@ -0,0 +1,267 @@
+{% extends "ceph/base_ceph.html" %}
+
+{% block title %}Ubuntu Pro for AWS{% endblock %}
+{% block meta_description %}Ubuntu Pro for AWS, the Ubuntu image optimized for production and professional use on public cloud. Including broad security coverage, live kernel patching, certified components with hardening profiles, and backed by a 10-years maintenance commitment by Canonical. Get it on the AWS marketplace.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1pLRWJZSpE04nYt-yHBtCjxGI7VZsigcP2UJIBmR8i-0{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-equal-height">
+      <div class="col-7 col-medium-4">
+        <h1>Ubuntu Pro for AWS</h1>
+        <p class="p-heading--three">For production. For professionals.</p>
+        <p>Ubuntu Pro for AWS is a premium image designed by Canonical to provide the most comprehensive feature set for production environments running in the public cloud. Offered as an AMI through AWS Marketplace, it includes a subscription to critical security and compliance services by default. Ubuntu Pro is suitable for small to large-scale Linux enterprise operations &mdash; no contract necessary.</p>
+        <p><a href="/aws/contact-us" class="p-button--positive js-invoke-modal">Launch Ubuntu Pro 18.04 LTS on AWS</a></p>
+        <p><a href="https://assets.ubuntu.com/v1/579f0c03-Ubuntu_Pro_AWS.pdf" class="p-link--external">Get the Ubuntu Pro datasheet</a></p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg" alt="" width="200">
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```

Reply:
I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.